main.py

```diff
@@ -174,10 +174,10 @@ def main():
         global supported_currencies
 
         if announcement_coin not in order and announcement_coin not in sold_coins and announcement_coin not in old_coins:
-            logger.info(f'New annoucement detected: {announcement_coin}')
-
-            if supported_currencies is not False:
-                if announcement_coin in supported_currencies:
+            if announcement_coin:
+                logger.info(f'New annoucement detected: {announcement_coin}')
+            if supported_currencies:
+                if announcement_coin is not False and announcement_coin in supported_currencies:
```

Comment:
I think it will work the same `if announcement_coin and announcement_coin in supported_currencies:`

Reply:
You are correct. I pushed another commit with this change